### PR TITLE
docs: update README.md to fix the "TypeError" issue in the "Feed Data" section

### PR DESCRIPTION
**Pull request by hongchhe**
Update README.md to fix the issue of "TypeError: 'builtin_function_or_method' object is not subscriptable" in the "Feed Data" section.

### CODE REVIEW

**Comment by JoanFM** [comment]
Thanks for your contribution. Unfortunately, as you didn't set up your local git config correctly, the author of your commit does not match with your Github account. This poses problems:
Your contribution can not be credited, and will not show up in the list of contributors (as bad git config = anonymous)
We can not accept CLA for anonymous contribution.
Please note that this does not only affect your contribution to Jina, but also affects all your contribution to anything on Github. So better fix it, once for all.
Here is what you can do:
```
git config user.name "Correct new name"
git config user.email "correct@new.email" -> match with your Github registered email address
```
Then fix the commit message by
```
git commit --amend --author="Author Name <email@address.com>" --no-edit
git push --force
```

**Comment by JoanFM** [comment]
Bad commit message, please refer to:
- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
#553 :bookmark: Commit Message Guideline for the First Time Contributor
jina-ai/jina | Jun 21st | Added by GitHub
CONTRIBUTING.md
jina-ai/jina | Added by GitHub

**Review by JoanFM — CHANGES_REQUESTED** [review]
Hello @hongchhe,

Thank you for your contribution, very good catch!

I do not see why we need to change the formatting of our Dockerfiles? However, if needed, we would appreciate if you could open different PRs to merge changes related to 2 so much different concepts.

File: README.md [hunk 1]
```diff
@@ -132,9 +132,10 @@ Let's create some random data and index it:
 
 ```python
 from jina import Document
+import numpy
```
[the remark this answers is not in the document]
Comment (hongchhe) [reply README.md:135]: Yes. Thanks for your review.

**Comment by hongchhe** [comment]
> Thanks for your contribution. Unfortunately, as you didn't set up your local git config correctly, the author of your commit does not match with your Github account. This poses problems:
> Your contribution can not be credited, and will not show up in the list of contributors (as bad git config = anonymous)
> We can not accept CLA for anonymous contribution.
> Please note that this does not only affect your contribution to Jina, but also affects all your contribution to anything on Github. So better fix it, once for all.
> Here is what you can do:
> 
> ```
> git config user.name "Correct new name"
> git config user.email "correct@new.email" -> match with your Github registered email address
> ```
> 
> Then fix the commit message by
> 
> ```
> git commit --amend --author="Author Name <email@address.com>" --no-edit
> git push --force
> ```

**Comment by hongchhe** [comment]
> Hello @hongchhe,
> 
> Thank you for your contribution, very good catch!
> 
> I do not see why we need to change the formatting of our Dockerfiles? However, if needed, we would appreciate if you could open different PRs to merge changes related to 2 so much different concepts.

Thanks for your detailed reminders and suggestions. It's a suggestion for the style of those Dockerfile changes due to I think it's more clearly and I will split it into the new PR for review. Thanks

**Comment by JoanFM** [comment]
> > Hello @hongchhe,
> > Thank you for your contribution, very good catch!
> > I do not see why we need to change the formatting of our Dockerfiles? However, if needed, we would appreciate if you could open different PRs to merge changes related to 2 so much different concepts.
> 
> Thanks for your detailed reminders and suggestions. It's a suggestion for the style of those Dockerfile changes due to I think it's more clearly and I will split it into the new PR for review. Thanks

could you upload a screenshot showing the differences of clarity when u do the PR? It is not so obvious when looking at the diffs here

**Comment by hongchhe** [comment]
Close this PR at first.

**Pull request closed without merging**